lms/djangoapps/commerce/signals.py

```diff
@@ -3,24 +3,16 @@
 """
 from __future__ import unicode_literals
 
-import json
 import logging
-from urlparse import urljoin
 
-import requests
-from django.conf import settings
-from django.contrib.auth import get_user_model
 from django.contrib.auth.models import AnonymousUser
 from django.dispatch import receiver
-from django.utils.translation import ugettext as _
 
 from entitlements.signals import REFUND_ENTITLEMENT
-from openedx.core.djangoapps.commerce.utils import ecommerce_api_client, is_commerce_service_configured
-from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
-from openedx.core.djangoapps.theming import helpers as theming_helpers
+from openedx.core.djangoapps.commerce.utils import is_commerce_service_configured
 from request_cache.middleware import RequestCache
 from student.signals import REFUND_ORDER
-from .models import CommerceConfiguration
+from .utils import refund_entitlement, refund_seat
```

Comment:
Just checking, this only gets added here because these methods are moved to utils?

Reply:
Yes